torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py

```diff
@@ -66,42 +66,22 @@ def get_supported_symmetric_config_and_operators() -> List[OperatorConfig]:
     return copy.deepcopy(supported_config_and_operators)
 
 
-def get_default_symmetric_qnnpack_quantization_config():
+def get_symmetric_quantization_config(is_per_channel=False):
     act_quantization_spec = QuantizationSpec(
         dtype=torch.int8,
         quant_min=-128,
         quant_max=127,
-        qscheme=torch.per_tensor_affine,
-        is_dynamic=False,
-    )
-    weight_quantization_spec = QuantizationSpec(
-        dtype=torch.int8,
-        quant_min=-127,
-        quant_max=127,
         qscheme=torch.per_tensor_symmetric,
```

Comment:
@kimishpatel @jerryzh168 This PR changed the activation qscheme from `per_tensor_affine` to `per_tensor_symmetric`. Is this intended? I don't think this matches the corresponding qconfig anymore (https://github.com/pytorch/pytorch/blob/6d5040a1ace7583862992bd1a1b606e873557fe1/torch/ao/quantization/qconfig.py#L288, which uses `per_tensor_affine`)